docs/en/journal.md

```diff
@@ -94,6 +94,24 @@ The parameter id in the route is the id in the URL of the user ’s Google Schol
 
 </RouteEn>
 
+## JASA
+
+### Latest Research
+
+<RouteEn author="Derekmini auto-bot-ty" example="/jasa/latest" path="/jasa/latest" radar="1" rssbud="1">
+
+</RouteEn>
+
+### Section Research
+
+<RouteEn author="Derekmini" example="/jasa/section/ANIMAL+BIOACOUSTICS" path="/jasa/section/:id" :paramsDesc="['Section Name，get it from tocSection of the URL']" radar="1" rssbud="1">
```

Comment:
```suggestion
<RouteEn author="Derekmini" example="/jasa/section/ANIMAL+BIOACOUSTICS" path="/jasa/section/:id" :paramsDesc="['Section Name, get it from tocSection of the URL']" radar="1" rssbud="1">
```

Reply:
Thanks for your attention! I have fixed it.